CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 5.0 (to be released)
 
 Removed features deprecated in [4.6](#4.6)
+Drop support for php 7.2, 7.3
```

Comment:
Please move it to a new 4.10 section, it looks like dropping support of EOL software is not a breaking change from the viewpoint of semver: https://www.reddit.com/r/Python/comments/107g04w/would_you_increment_semver_when_removing_support/

(and I see that we forgot to mention 4.8 and 4.9 in the changelog :see_no_evil:)

Reply:
I changed it to your suggestions, and fixed the anchor-links. 
It is not a BC change, but noteable for a changelog.

It is like always in software development. Documentation is not the priority ;)